typings/index.d.ts

```diff
@@ -1425,6 +1427,23 @@ declare module 'discord.js' {
     public static resolve(bit?: BitFieldResolvable<SpeakingString, number>): number;
   }
 
+  export class StageChannel extends GuildChannel {
+    public readonly members: Collection<Snowflake, GuildMember>;
+    public readonly editable: boolean;
+    public readonly full: boolean;
+    public readonly joinable: boolean;
+    public readonly speakable: boolean;
```

Comment:
these should be moved to underneath the constructor, to keep consistent with the order in other classes